crates/model/src/order.rs

```diff
@@ -614,6 +614,8 @@ pub struct OrderMetadata {
     /// Does not take partial fill into account.
     #[serde(default, with = "u256_decimal")]
     pub solver_fee: U256,
+    #[serde(default, with = "u256_decimal")]
+    pub executed_solver_fee: U256,
```

Comment:
This feels like it should be optional. Not all orders have it, and I'd rather have an explicit None than implicit 0.

Reply:
Resolved.